File: JavaScript/프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = new Array(2);
    answer[0]=0;
    answer[1]=0;
    for(var i=0; i<num_list.length ; i++){
        if(num_list[i]%2==0) {
            answer[0]++;
        }
        else answer[1]++;
    }
    return answer;
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.cs
using System;

public class Solution {
    public int[] solution(int[] num_list) {
        int oddcount=0;
        int evencount=0;
        for(int i=0; i<num_list.Length;i++){
            if(num_list[i]%2==0) evencount++;
            else oddcount++;
        }
        int[] answer = {evencount, oddcount};
        return answer;
    }
}